src/lib/input/input.scss

```diff
@@ -95,6 +95,10 @@ md-input {
   }
 }
 
+.md-input-element-textarea {
+  min-height: 60px;
+}
```

Comment:
Where does this 60px come from? I don't see anything in the spec and it looks like material1 uses a min-height to match the line-height. Can we remove it?